httpie/models.py

```diff
@@ -61,12 +61,27 @@ def headers(self):
             20: '2',
         }[original.version]
 
-        status_line = f'HTTP/{version} {original.status} {original.reason}'
+        return version, original.status, original.reason, original.msg._headers
```

Comment:
Status, reason and headers ought to be pulled from self._orig as is done in _headers_requests_response instead of accessing the protected member self._orig.raw._original_response. Its not clear why this was not done to begin with and for that reason I'm leaving the old logic in place.

Notable that same approach cannot be used for version. Perhaps it would be better to file a bug against requests and ask for HTTP version to be exposed without having to rely on protected members.

Reply:
If its reasonable to assume that
```
status = self._orig.raw._original_response.status_code
reason = self._orig.raw._original_response.reason
headers = self._orig.raw._original_response.msg._headers
```
Is effectively the same as.
```
status = self._orig.status_code
reason = self._orig.reason
headers = self._orig.headers.items()
```

Then its worthwhile to consider the slightly simpler implementation https://github.com/IlyaSukhanov/httpie/commit/c14deea1a68c979f7700be1217512ee39c06d742

Reply:
I would be in favor of the slightly simpler implementation but I would like @jakubroztocil POV, if possible.

Reply:
Sounds good. Let me know if you'd prefer I open that as a separate pr or include it on this one.

Reply:
Let's go with the simplified version :+1:

Reply:
Done